corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -85,7 +85,6 @@ def test_get_descendants_with_empty_queryset(self):
         self.assertEqual(locs.count(), 0)
 
     def test_getitem_with_slice(self):
-        # remove this test when removing ComparedQuerySet
```

Comment:
Did this not mean "remove the whole test"?

Reply:
It did, but I see no reason to remove it now since it seems like a reasonable test and there is untested code somewhere that exercises this feature. Maybe I'm being too conservative...